google/cloud/storage/client.py

```diff
@@ -836,6 +846,165 @@ def get_hmac_key_metadata(
         metadata.reload(timeout=timeout)  # raises NotFound for missing key
         return metadata
 
+    def generate_signed_post_policy_v4(
+        self,
+        credentials,
```

Comment:
Right now credentials is a positional argument and should be optional like generate_signed_url -> https://github.com/googleapis/python-storage/blob/a50cdd145d39cb67e0d7021c376acbe376436178/google/cloud/storage/blob.py#L363

Reply:
Done